product_examples/errors-and-omissions/products/personal-auto/policy/exposures/vehicle/exposure.json

```diff
@@ -0,0 +1,256 @@
+{
+  "name": "vehicle",
+  "displayName": "Vehicle",
+  "fields": [
+    {
+      "name": "vehicle_type",
+      "title": "Vehicle Type",
+      "type": "select",
+      "width": 3,
+      "values": [
+        "Car",
+        "SUV",
+        "Pickup",
+        "Motorcycle"
+      ]
+    },
+    {
+      "name": "body_style",
+      "title": "Body Style",
+      "type": "select",
+      "width": 3,
+      "values": [
+        "2 door",
+        "4 door"
+      ],
+      "condition": {
+        "vehicle_type": [
+          "Car"
+        ]
+      }
+    },
+    {
+      "name": "motorcycle_cc",
+      "title": "CC",
+      "width": 3,
+      "type": "number",
+      "condition": {
+        "vehicle_type": [
+          "Motorcycle"
+        ]
+      }
+    },
+    {
+      "name": "vin",
+      "title": "Vehicle Identification Number (eg  2HGEJ6526VH515413  )",
+      "type": "string",
+      "newline": true,
+      "optional": true
+    },
+    {
+      "newline": true,
+      "name": "year",
+      "title": "Vehicle Year",
+      "type": "string",
+      "regex": "^(19[3-9][0-9]|2[0-9]{3})$",
+      "width": 3
+    },
+    {
+      "name": "make",
+      "title": "Vehicle Make",
+      "type": "string",
+      "width": 3
+    },
+    {
+      "name": "model",
+      "title": "Vehicle Model",
+      "type": "string",
+      "width": 3
+    },
+    {
+      "newline": true,
+      "name": "vehicle_value",
+      "title": "Vehicle Value",
+      "type": "number",
+      "width": 3,
+      "decimalPlaces": 0
+    },
+    {
+      "name": "unrepaired_damage",
+      "title": "Unrepaired Damage",
+      "width": 3,
+      "optional": true,
+      "type": "select",
+      "values": [
+        "No - Existing Damage",
+        "Yes - Existing Damage"
+      ]
+    },
+    {
+      "name": "ownership_indicator",
```

Comment:
The use of 'Indicator' here seems inconsistent. Maybe remove that word, just make it 'Ownership?' (Each field value here is an Indicator, but this is the only value we include that word for)